### PR TITLE
Add env var to --help output for flags and args

### LINKED ISSUE #270: A way to show binded environment variables in the help text
Hey there!

It would be nice if we will have an option/flag to show binded `Envar`'s in the help text (probably for `--help-long` or `--help-man`).
Something like this:
```
-i, --interval=0      set commit interval (From ENV MY_APP_COMMIT_INTERVAL)
```

Thoughts ?

### PR DESCRIPTION
**Pull request by autarch**
Fixes #270

### CODE REVIEW

File: model.go [hunk 1]
```diff
@@ -90,6 +90,13 @@ func (f *FlagModel) FormatPlaceHolder() string {
 	return strings.ToUpper(f.Name)
 }
 
+func (f *FlagModel) HelpWithEnvar() string {
+	if f.Envar == "" {
+		return f.Help
+	}
+	return fmt.Sprintf("%s (env var: %s)", f.Help, f.Envar)
```
Comment (alecthomas) [inline model.go]: Instead of `(env var: %s)` just make this `($%s)`, it's shorter and still clear.

File: model.go [hunk 2]
```diff
@@ -114,6 +121,13 @@ func (a *ArgGroupModel) ArgSummary() string {
 	return strings.Join(out, " ")
 }
 
+func (a *ArgModel) HelpWithEnvar() string {
+	if a.Envar == "" {
+		return a.Help
+	}
+	return fmt.Sprintf("%s (env var: %s)", a.Help, a.Envar)
```
Comment (alecthomas) [inline model.go]: Same here.

File: model.go (hunk 1 above)
Comment (autarch) [reply model.go]: Done and rebased.

File: model.go (hunk 2 above)
Comment (autarch) [reply model.go]: Done and rebased.

**Comment by aksdb** [comment]
@alecthomas Any chance you have a few minutes to review this again? I would like this feature as well :-)

**Pull request merged**